src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -86,11 +86,26 @@
       [:div {:style {:padding "1em 0 0 1em" :fontWeight 600}}
        (get-in props [:data :name])]])})
 
+(defn- get-workspace-name-string [ws]
+  (str (get-in ws ["workspace" "namespace"]) "/" (get-in ws ["workspace" "name"])))
+
+(defn- get-max-workspace-name-length [workspaces]
+  (apply max (map count (map #(get-workspace-name-string %) workspaces))))
+
+(defn- get-workspace-description [ws]
+  (get-in ws ["workspace" "attributes" "description"]))
```

Comment:
(not a full code review, but ...) is there any way to integrate get-workspace-description and get-workspace-name-string with the renders in WorkspaceTable, so we're not duplicating logic? (i.e. "if no description entered, use 'No description provided'")